kivy/graphics/vertex_instructions_line.pxi

```diff
@@ -1310,6 +1355,11 @@ cdef class SmoothLine(Line):
         VertexInstruction.apply(self)
         return 0
 
+    # FIXME: Some artifacts can be observed, depending on the line width,
+    # overdraw_width and radius. The minimum width and/or height of
+    # rounded_rectangle also needs to be limited to 2px.
```

Comment:
Is this now addressed? As the doc says that it won't render https://github.com/kivy/kivy/pull/7876/files#diff-2e9fcb864c08a6bf13e14b06a577a4f19f1de1cc6274381f03112eea4b854442R1117 the radius = 0 issue seems to be addressed as well (comment + code change).

Reply:
Updated. Would it be important to keep this note that smoothline will not render at 2px dimensions?